### PR TITLE
Add helper methods to lookup value/commandclasses

### PR DESCRIPTION
**Pull request by marcelveldt**
As discussed before I've added a few helper methods which allows us to check if a node(instance) contains a specific commandclass or value.

### CODE REVIEW

File: openzwavemqtt/models/command_class.py [hunk 1]
```diff
@@ -50,3 +51,15 @@ def time_stamp(self) -> int:
     def create_collections(self):
         """Create collections that Node supports."""
         return {"value": ItemCollection(OZWValue)}
+
+    def get_value(self, value_index: ValueIndex) -> bool:
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/command_class.py:55]: The return value should be an optional value type.

File: openzwavemqtt/models/node.py [hunk 2]
```diff
@@ -220,3 +226,31 @@ def create_collections(self):
             "association": ItemCollection(OZWNodeAssocation),
             "statistics": OZWNodeStatistics,
         }
+
+    def get_command_class(
+        self, command_class_id: CommandClass, instance_id=None
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/node.py:231]: `instance_id` is missing typing.

File: openzwavemqtt/models/node.py [hunk 3]
```diff
@@ -220,3 +226,31 @@ def create_collections(self):
...
+    ) -> bool:
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/node.py:232]: The return value typing should not be bool.

File: openzwavemqtt/models/node.py [hunk 4]
```diff
@@ -220,3 +226,31 @@ def create_collections(self):
...
+        """Return a specific CommandClass on this node (if exists)."""
+        # pylint: disable=no-member
+        for instance in self.instances():
+            if not (instance_id is None or instance.instance == instance_id):
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/node.py:236]: It's easier to read if we remove the parenthesis and adjust the expression accordingly.

File: openzwavemqtt/models/node.py [hunk 5]
```diff
@@ -220,3 +226,31 @@ def create_collections(self):
...
+
+    def has_command_class(
+        self, command_class_id: CommandClass, instance_id=None
+    ) -> bool:
+        """Determine if the node has the given CommandClass."""
+        return self.get_command_class(command_class_id, instance_id) is not None
+
+    def get_value(
+        self, command_class_id: CommandClass, value_index: ValueIndex, instance_id=None
+    ) -> bool:
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/node.py:249]: Return value typing.

File: openzwavemqtt/models/node_instance.py [hunk 6]
```diff
@@ -29,3 +31,28 @@ def time_stamp(self) -> int:
     def create_collections(self):
         """Create collections that Node supports."""
         return {"commandclass": ItemCollection(OZWCommandClass)}
+
+    def get_command_class(self, command_class_id: CommandClass) -> bool:
```
Comment (MartinHjelmare) [inline openzwavemqtt/models/node_instance.py:35]: Please check the typing for these helpers too.

File: openzwavemqtt/models/command_class.py (hunk 1 above)
Comment (marcelveldt) [reply openzwavemqtt/models/command_class.py:55]: yup, sloppy copy/paste work.

**Pull request merged**